### PR TITLE
naval jacket on more naval maps

### PR DESCRIPTION
**Pull request by Crackaduck**
## About the PR 
puts the hops naval jacket on oshan, clarion, horizion, and destiny

## Why's this needed? 
the naval jacket, being the hops only jacket, should be on more maps to give them more options. While other outfits got this treatment, the hops did not. A previous PR i made, saw the hop's coat be added to all maps, however was looked upon less faborably. This PR adds it to more maps, but makes it so its still locked to maps upon which it only makes sense, water based maps, or maps which are ships.

## Changelog

```changelog
(u)MozErella
(+)Hops naval jacket appears on more maps.
```

### CODE REVIEW

File: code/datums/jobs.dm [hunk 1]
```diff
@@ -215,7 +215,7 @@ ABSTRACT_TYPE(/datum/job/command)
 	announce_on_join = 1
 
 
-#ifdef SUBMARINE_MAP
+#ifdef FLEET_MAP
```
Comment (ZeWaka) [inline code/datums/jobs.dm:218]: having mixed normal `ifdef` and value-associated `define`s is gross, should just choose one

File: _std/mapDefines.dm [hunk 2]
```diff
@@ -3,9 +3,11 @@
 
 #elif defined(MAP_OVERRIDE_DESTINY)
 
+#define FLEET_MAP 1
```
Comment (ZeWaka) [inline _std/mapDefines.dm:6]: `fleet` is poor naming, `ship` would be more clear

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: code/datums/jobs.dm (hunk 1 above)
Comment (Crackaduck) [reply code/datums/jobs.dm:218]: so the request changes thing, how would i allow that or is that supposed to be a "go in and fix it" thing

Comment (ZeWaka) [reply code/datums/jobs.dm:218]: you need to fix it.

Comment (Crackaduck) [reply code/datums/jobs.dm:218]: i dunno what you really mean by that, the ifdef part. thats the way it normally is and i replaced it with fleet

Comment (Crackaduck) [reply code/datums/jobs.dm:218]: as in, how would i change this to be more suitable for you?

Comment (ZeWaka) [reply code/datums/jobs.dm]: you gate with an ifdef but assign a value of 1 that you never check

**Pull request closed without merging**